llvm/lib/Passes/PassRegistry.def

```diff
@@ -590,6 +590,10 @@ FUNCTION_PASS_WITH_PARAMS(
       return WinEHPreparePass(DemoteCatchSwitchPHIOnly);
     },
     parseWinEHPrepareOptions, "demote-catchswitch-only")
+FUNCTION_PASS_WITH_PARAMS(
+    "rtsan", "RealtimeSanitizerPass",
```

Comment:
are these parameters expected in future?
If not, you can use simpler macro.

Reply:
Yes, we are expecting to add options very soon, which is why I left it with the parameters (and the "empty" parameter file).